run.sh

```diff
@@ -85,24 +85,25 @@ else
 fi
 
 # Run reform.py
+# ./uploads/$timestamp/
 
 if [ ! -z "$position" ]; then
-  echo   /home/reform/venv/bin/python reform.py --chrom $chrom --position $position --in_fasta ./uploads/$timestamp/$in_fasta \
-  --in_gff ./uploads/$timestamp/$in_gff --ref_fasta "$ref_fasta_path" --ref_gff "$ref_gff_path" \
+  echo   /home/reform/venv/bin/python reform.py --chrom $chrom --position $position --in_fasta $in_fasta \
```

Comment:
`/home/reform/venv/bin/python` is bad form. I see it as there originally, but we should have relative paths instead. Especially now that we have a dev user which won't have access to /home/reform.